tests/testdata/codegen/output-funcs/nodejs-extras/tests/codegen.spec.ts

```diff
@@ -17,6 +17,11 @@ import * as assert from "assert";
 import * as pulumi from "@pulumi/pulumi";
 import * as sut from "..";
 
+function createUnknown() {
```

Comment:
We export this publicly for C# for testing purposes, maybe we should just export this for node as well?

Reply:
Yeah I don't see a reason why not, will remove the `@internal` flag